cell_type_prediction/galaxy_cli_use_case/inputs.yaml

```diff
@@ -0,0 +1,12 @@
+expression_matrix:
```

Comment:
I would rename this file to <current-name>.template, add place holders in the field like `<GENES_PATH>` so that then it can be used as a template where you replace values with (and show a way to do it, like with sed). In time we could move the argument parsing to click I guess and enable the cli to override fields of these files.